### PR TITLE
goobook: add page

### PR DESCRIPTION
**Pull request by dinaisrael**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/goobook.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# goobook
+
+> Access your Google contacts from mutt or the command line.
```
Comment (kbdharun) [inline pages/linux/goobook.md:3]:
```suggestion
> Access Google contacts from `mutt` or the command line.
```

File: pages/linux/goobook.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://manned.org/goobook>.
+
+- Allow goobook to access your Google contacts using OAuth2:
```
Comment (kbdharun) [inline pages/linux/goobook.md:6]:
```suggestion
- Allow goobook to access Google contacts using OAuth2:
```

File: pages/linux/goobook.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`goobook authenticate`
+
+- Dump all your contacts to XML (stdout):
```
Comment (kbdharun) [inline pages/linux/goobook.md:10]:
```suggestion
- Dump all contacts to XML (stdout):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by dinaisrael** [comment]
Thank you @kbdharun :) I've committed the suggestions

File: pages/linux/goobook.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# goobook
+
+> Access Google contacts from `mutt` or the command line.
+> More information: <https://manned.org/goobook>.
+
+- Allow goobook to access Google contacts using OAuth2:
```
Comment (kbdharun) [inline pages/linux/goobook.md:6]:
```suggestion
- Allow `goobook` to access Google contacts using OAuth2:
```
Although we don't recommend using the command name in command descriptions. I guess this one fine.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**